spec/models/survey_section_spec.rb

```diff
@@ -261,16 +265,16 @@
         :answer_id => mixed_mandatory_answer_2.id,
         :string_value => "answer"
       )
-      new_set.save.should be true
-      dependency_section.completed?( new_set ).should be false
+      new_set.save!
+      expect(dependency_section.completed?(new_set)).to be(false)
 
       new_set.responses.build(
         :question_id => dependency_mandatory_question.id,
         :answer_id => dependency_mandatory_answer.id,
         :string_value => "answer"
       )
-      new_set.save.should be true
-      dependency_section.completed?( new_set ).should be true
+      new_set.save!
+      expect(dependency_section.completed?(new_set)).to be(true)
```

Comment:
how did these get missed? thanks for catching it

Reply:
There are a few more -- these were just in the files I was playing with when trying to figure out what was going on. I don't think this actually breaks rspec 3, it just gets sad. We likely should go through and get all of them but this was mostly just clean up that we didn't _need_ but I didn't want to lose.

Reply:
I thought my branch did all of this  originally.... there's a 2->3 upgrade tool